### PR TITLE
Upgrade triggers to v0.13.0

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

File: crds/tekton-triggers.yaml [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterinterceptors.triggers.tekton.dev
```
Comment (fiunchinho) [inline crds/tekton-triggers.yaml:19]: There is a new CRD added on this release

File: tekton/triggers/interceptors.yaml [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# Copyright 2020 The Tekton Authors
```
Comment (fiunchinho) [inline tekton/triggers/interceptors.yaml:1]: These are the core interceptors. They got moved out of the main `release.yaml` file into its own file.

File: tekton/triggers/upstream.yaml [hunk 3]
```diff
@@ -775,105 +741,6 @@ spec:
             # User 65532 is the distroless nonroot user ID
             runAsUser: 65532
 
----
```
Comment (fiunchinho) [inline tekton/triggers/upstream.yaml:778]: The core interceptors got moved to its own file on this release.

File: tekton/triggers/upstream.yaml [hunk 4]
```diff
@@ -23,23 +23,23 @@ spec:
   privileged: false
   allowPrivilegeEscalation: false
   volumes:
-    - "emptyDir"
-    - "configMap"
-    - "secret"
+    - 'emptyDir'
```
Comment (MarcelMue) [inline tekton/triggers/upstream.yaml:26]: Only weird thing is the quotation marks here.

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**